app/src/main/java/com/wednesday/bankingdetails/activity/MainActivity.kt

```diff
@@ -17,8 +22,8 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        val bankListAdapter: BankListAdapter
-        val bankList: ArrayList<Bank> = BankFactory.getMockBanks()
+        var bankListAdapter: BankListAdapter
```

Comment:
This should not be a var. It is only initialized once. Use lazy variables.

Reply:
Done. Used lazy variables.